runtime/src/iree-amd-aie/aie_runtime/AMDAIEEnums.h

```diff
@@ -11,9 +11,27 @@
 #define IREE_AIE_RUNTIME_AMDAIE_ENUMS_H_
 
 #include "mlir/IR/BuiltinAttributes.h"
-
 // clang-format off
 #include "iree-amd-aie/aie_runtime/AMDAIEEnums.h.inc"
 // clang-format on
 
+// TODO(someone_who_knows) explain the connection between npu numbering and
+// aie numbering, and why npu jumps from '1' to '4'.
```

Comment:
Because npu2/3 were already used in xdna-driver for some custom devices, one of them might be a 4x4 variation of Strix and not sure about the other one. Npu2 seems to have been the prototype for Strix (maybe the 4x4 variation)?

Reply:
Thanks, incorporated